modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Numeric/Numeric.es.js

```diff
@@ -25,14 +25,21 @@ class Numeric extends Component {
 	applyMask() {
 		const {dataType, element} = this;
 		const inputElement = element.querySelector('input');
+		const value = inputElement.value;
 		const numberMaskOptions = this.getMaskConfig(dataType);
 
+		if (value && dataType == 'integer') {
+			inputElement.value = Math.round(value.replace(',', '.'));
```

Comment:
@alinedoleron is there a reason for using hardcoded comma instead of the symbol variable?